convert2rhel/toolopts.py

```diff
@@ -373,11 +394,21 @@ def _process_cli_options(self):
         if parsed_opts.pool:
             tool_opts.pool = parsed_opts.pool
 
+        if parsed_opts.activationkey:
+            tool_opts.activation_key = parsed_opts.activationkey
+
+        if parsed_opts.org:
+            tool_opts.org = parsed_opts.org
+
         if parsed_opts.serverurl:
             if tool_opts.no_rhsm:
                 loggerinst.warning(
                     "Ignoring the --serverurl option. It has no effect when --disable-submgr or --no-rhsm is used."
                 )
+            elif not tool_opts.should_subscribe:
+                loggerinst.warning(
+                    "Ignoring the --serverurl option. It has no effect when no credentials to subscribe the system were given."
+                )
```

Comment:
If we deprecate the `no_rhsm` option, we will just remove the above statement and use this one? 

I'm wondering if it is not easier to merge the two issues right now and remove the `no_rhsm` option.

Reply:
<nod> Yeah, I' working on adding norhsm into the mix now, then working on unittests afterthat.

Reply:
So actually, it is `--keep-rhsm` that is being deprecated, not `--no-rhsm`.  So the above code is still present.